src/js/components/commits/list.tsx

```diff
@@ -67,7 +67,7 @@ const MessageTableCell = ({
 }: TableCellProps) => (
   <DataTableCell
     {...props}
-    className={classNames(className, 'commits-message')}
+    className={classNames(className, 'commits-message', 'truncated-cell')}
```

Comment:
I'm not sure we are using `commits-message` for anything now, are we?

Reply:
I suppose we aren't using the `commits-author` class either, but I really wouldn't mind keeping both of these in here since it is hard to add them back in (for me at least) if we do need them in the future.

Reply:
Right. We're not using some of these, but I left them around anyway.